models.go

```diff
@@ -23,7 +22,7 @@ type Team struct {
 type Account struct {
 	Name     string `json:"name"`
 	RoleArn  string `json:"roleArn"`
-	Duration int    `json:"duration"`
+	Duration int64  `json:"duration"`
 }
 
 func (a *Account) UnmarshalJSON(b []byte) error {
```

Comment:
I don't think we should implement a custom `UnmarshalJSON` to do validation; instead I think it would make sense to put this functionality in the `AssumeRole` method for instance - so if `duration` is 0 (the default value for `int64`) we know that duration was not set by the user, and we can just default to 3600?

Reply:
I added the logic to AssumeRole as you suggested.